scout/server/blueprints/variants/templates/variants/variants.html

```diff
@@ -71,7 +71,7 @@
             {% for variant in variants %}
               {% if variant.dismiss_variant %}
                 <tr class="dismiss">
-              {% elif variant._id in case.causatives %}
+              {% elif causatives in case and variant._id in case.causatives %}
```

Comment:
This one!

Reply:
Got there at the same time, but thanks! 😆

Reply:
...awaiting the tests with some tension...

Reply:
Nooo we're good! 🌴

Reply:
🍹🍉 ...